tests/test-sources/plugins/utils/which-key.nix

```diff
@@ -251,6 +251,31 @@
               desc = "Vim cmd KeyMapping Test";
               inherit mode;
             }
+            # Nested mapping
+            {
+              mode = [
+                "n"
+                "v"
+              ];
+              __unkeyed-1 = [
+                {
+                  __unkeyed-1 = "<leader>f";
+                  group = "Group Test";
+                }
+                {
+                  __unkeyed-1 = "<leader>f<tab>";
+                  group = "Group in Group Test";
+                }
+              ];
+            }
+            # Create mapping
+            {
+              __unkeyed-1 = "<leader>cS";
+              __unkeyed-2 = ":CodeSnapSave<CR>";
```

Comment:
This _really_ isn't important here, as it's just a test example, but any `:foo<cr>` mapping can be replaced with `<cmd>foo<cr>` with a few caveats:
- it must end with `<cr>`
- it is automatically silent

Using `<cmd>` will work better than `:` when the mapping is used in some modes

Reply:
Good to know, thank!